clients/filesystem-hadoop3/src/main/java/com/datastrato/gravitino/filesystem/hadoop/GravitinoVirtualFileSystem.java

```diff
@@ -51,6 +53,9 @@ public class GravitinoVirtualFileSystem extends FileSystem {
   private Cache<NameIdentifier, Pair<Fileset, FileSystem>> filesetCache;
   private ScheduledThreadPoolExecutor scheduler;
 
+  private static final Pattern IDENTIFIER_PATTERN =
```

Comment:
Could you add a description for this pattern which string will be matched?

Reply:
done